index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Jerry && Faith</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="main">
+        <div id="title">
+          <h1>Dr Norman Borlaug</h1>
+          <h4>The man who saved a billion lives</h4>
+        </div>
+
+    </div>
+    <figure id="img-div">
+      <img id="image" src="./images/tribute-page-main-image.jpg" alt="">
+      <figcaption id="img-caption">
+        Dr. Norman Borlaug, third from the left, trains biologists in 
+        Mexico on how to increase wheat yields - part of his life-long war on hunger.
+      </figcaption>
+    </figure>
+    <div id="tribute-info">
+      <h2>Here's a time line of Dr. Borlaug's life:</h2>
+      <!-- <p><strong></strong> -  Born in Cresco, Iowa </p> -->
+      <ol>
+        <li><strong>1914</strong> - Born in Cresco, Iowa</li>
+        <br>
+        <li><strong>1933</strong> - Leaves his family's farm to attend the University of Minnesota, thanks 
+          to a Depression era program known as the "National Youth Administration"</li>
+        <br>
+        <li>1935<strong></strong>- Has to stop school and save up more money. Works in the Civilian 
+            Conservation Corps, helping starving Americans. "I saw how food changed them", he said. 
+            "All of this left scars on me."</li>
+          <br>
+        <li>1939<strong></strong>- Has to stop school and save up more money. Works in the Civilian 
+          Conservation Corps, helping starving Americans. "I saw how food changed them", he said. 
```

Comment:
you made an omission on line 38. You probably want to move year into <strong> tags.

Reply:
- [ ] Kindly make sure to explain further the omission on line 38 as well as explaining the possible ways of correcting the issue, this will better help the student to understand the reason for the change and implement best practices. :+1: